www/front_src/src/Resources/Details/tabs/Details/DetailsCard/cards.tsx

```diff
@@ -121,11 +124,19 @@ const getDetailCardLines = ({
       line: <DetailsLine line={toDateTime(details.last_status_change)} />,
       title: labelLastStateChange,
     },
+    {
+      field:
+        details.last_time_with_no_issue &&
+        details.status.severity_code === SeverityCode.Ok,
```

Comment:
```suggestion
        equals(details.status.severity_code, SeverityCode.Ok),
```
You can use equals function from Ramda :)

Reply:
Yes it's not false :) it's better by ramda